lib/utils/tests-options-validator.js

```diff
@@ -132,7 +132,7 @@ module.exports = class TestsOptionsValidator {
     }
 
     // The parallel option accepts a number, which can be 0
-    if (this.options.parallel !== undefined) {
+    if (typeof this.options.parallel !== 'undefined') {
```

Comment:
im curious, what motivated this change?

Reply:
just to unify things.. in other if block we are using `typeof` for other options so make all the same :)